docker/src/main/docker/postgresql/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM avatica-server:latest
+MAINTAINER Apache Avatica <dev@calcite.apache.org>
+
+# Dependencies
+ADD https://repo1.maven.org/maven2/org/postgresql/postgresql/42.0.0.jre7/postgresql-42.0.0.jre7.jar /home/avatica/classpath/
```

Comment:
42.0.0.jre7 was the most recent one I could find. Pulled it out and will do some digging to update the website to list the major client version used (had that nagging thought in the back of my head anyways)